index.rst

```diff
@@ -98,6 +98,7 @@ The `README <https://github.com/lsst-dm/dm_dev_guide/blob/master/README.md>`__ w
    build-ci/lsstsw.rst
    build-ci/new_package.rst
    build-ci/third_party.rst
+   build-ci/github_protect_travis_flake8.rst
```

Comment:
`ci_overview.rst` was designed to be the root page for this. If @jonathansick decided that that page is not relevant we probably need to edit it to remove the Travis reference.